src/mango_opts.erl

```diff
@@ -192,6 +209,14 @@ validate_selector(Else) ->
     ?MANGO_ERROR({invalid_selector_json, Else}).
 
 
+%% We re-use validate_use_index to make sure the index names are valid
```

Comment:
This isn't a valid case since validate_use_index also takes into account forward slashes. You should do something like:

    lists:foreach(fun ?MODULE:is_string/1, Docs)